appeals/web/src/server/app/app.controller.js

```diff
@@ -15,19 +15,14 @@ import * as authSession from './auth/auth-session.service.js';
  *
  * @type {import('@pins/express').RenderHandler<ViewHomepageRenderOptions>}
  */
-export function viewHomepage(request, response) {
+export const viewHomepage = async (request, response) => {
```

Comment:
Is the `async` keyword needed here? I can't see any `await`s in the function body

Reply:
mmm, yeah left over from when I was testing the service... May actually leave it for the moment, I'm sure will be needed.